--- scripts/main/server.js	2022-04-09 05:02:50.938740555 +0200
+++ scripts/main/server_patched.js	2022-04-09 05:26:24.355275544 +0200
@@ -40,7 +40,7 @@
     // Build Main Server
     const app = express();
     const api = new PoolApi(_this.client, _this.poolConfigs, _this.portalConfig);
-    const limiter = rateLimit({ windowMs: 15 * 60 * 1000, max: 100 });
+    const limiter = rateLimit({ windowMs: 15 * 60 * 1000, max: 2000 });
     const cache = apicache.options({}).middleware;
 
     // Establish Middleware
